typing/ctype.ml

```diff
@@ -1365,7 +1357,7 @@ let rec copy_sep ~cleanup_scope ~fixed ~free ~bound ~may_share
               visited t1 in
           Tpoly (body, tl')
       | Tfield (p, k, ty1, ty2) -> (* the kind is kept shared *)
-          Tfield (p, field_kind_repr k, copy_rec ~may_share:true ty1,
+          Tfield (p, (*field_kind_repr*) k, copy_rec ~may_share:true ty1,
```

Comment:
Please don't leave dead code as comment :)

Reply:
Oops, this part was about to be restored later (for performance). I will soon push the update.

Reply:
Actually, re-enabling normalization here does not seem to improve performance in a measurable way, but at least it ensures that we don't keep long indirection chains inside copies (needed, as in-place path compression is only done for `type_expr`).